omod/src/main/java/org/openmrs/module/owa/filter/OwaFilter.java

```diff
@@ -66,6 +66,13 @@ public void doFilter(ServletRequest req, ServletResponse res, FilterChain chain)
 		}
 	}
 	
+	private boolean confirmRequestUrl(String requestURL) {
+		if (requestURL.contains("addonmanager")) {
+			return true;
+		}
+		return false;
+	}

```

Comment:
Have you tried to refactor this method to use Ternary operator
Something like:
`private boolean confirmRequest(string requestUrl){ requestUrl.contains('var') ? true : false}`

Reply:
Let me check it out

Reply:
If it dosen't work try this:
`private boolean confirmRequestUrl(String requestURL){
  boolean isRequesting = requestURL.contains("addonmanager") ? true : false;
  return 
}`